Source/WebCore/platform/graphics/gstreamer/eme/CDMOpenCDM.cpp

```diff
@@ -190,6 +190,7 @@ CDMInstanceOpenCDM::Session::Session(const media::OpenCdm& source, Ref<WebCore::
     : m_session(source)
     , m_initData(WTFMove(initData))
 {
+    // FIXME: This can be a source of overflows and other big errors. This should be fixed in the framework and then fixed here.
```

Comment:
I believe the framework is using the array size passed into to not overflow the caller's buffer. I agree the style is awful, but I was wondering what needs to be fixed that is currently a big error?

Reply:
From the WebKit pov we are giving a third party some memory to write on and we should not underestimage programmers ability to make mistakes and overflow our memory, which can lead to errors. That's why I'd prefer that the framework gave us some std::string with the value, for example.

Reply:
Right, just wondered if the framework is currently doing something bad with the passed memory (in which case we should file a report). I agree the interface needs changing.